packages/blade/src/components/Input/SelectInput/SelectInput.tsx

```diff
@@ -66,7 +66,8 @@ const _SelectInput = (
 
   React.useEffect(() => {
     onChange?.({ name: props.name, values: value.split(', ') });
-  }, [value, onChange, props.name]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [value, props.name]);
```

Comment:
Having `onChange` in this array was making the function trigger on parent re-renders which was causing infinite re-renders.

Reply:
[useEvent](https://github.com/reactjs/rfcs/blob/useevent/text/0000-useevent.md) hook might be useful here

Reply:
yup